phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/LibraryFeaturePanel.java

```diff
@@ -231,15 +231,18 @@ public void uninstallShortcut(ShortcutDTO shortcut) {
                 .withMessage(tr("Are you sure you want to uninstall {0}?", shortcutName))
                 .withOwner(getScene().getWindow())
                 .withResizable(true)
-                .withYesCallback(() -> getShortcutManager().uninstallFromShortcut(shortcut, e -> {
-                    final ErrorDialog errorDialog = ErrorDialog.builder()
-                            .withMessage(tr("Error while uninstalling {0}", shortcutName))
-                            .withException(e)
-                            .withOwner(getScene().getWindow())
-                            .build();
-
-                    errorDialog.showAndWait();
-                }))
+                .withYesCallback(() -> {
+                    getShortcutManager().uninstallFromShortcut(shortcut, e -> {
+                        final ErrorDialog errorDialog = ErrorDialog.builder()
+                                .withMessage(tr("Error while uninstalling {0}", shortcutName))
+                                .withException(e)
+                                .withOwner(getScene().getWindow())
+                                .build();
+
+                        errorDialog.showAndWait();
+                    });
+                    // TODO: how can the feature panel close itself?
```

Comment:
Do you mean the feature panel or the corresponding details panel?

Reply:
The corresponding details panel. It's a bit confusing because the buttons are on the details panel but the control is in the feature panel class.